test/Base.t.sol

```diff
@@ -35,12 +34,13 @@ import { SablierV2ProxyTargetPush } from "src/SablierV2ProxyTargetPush.sol";
 import { WLC } from "./mocks/WLC.sol";
 import { Assertions } from "./utils/Assertions.sol";
 import { Defaults } from "./utils/Defaults.sol";
+import { DeployPrecompiled } from "./utils/DeployPrecompiled.sol";
 import { Events } from "./utils/Events.sol";
 import { Merkle } from "./utils/Murky.sol";
 import { Users } from "./utils/Types.sol";
 
 /// @notice Base test contract with common logic needed by all tests.
-abstract contract Base_Test is Assertions, Events, Merkle, StdCheats, V2CoreUtils {
+abstract contract Base_Test is Assertions, DeployPrecompiled, Events, Merkle, V2CoreUtils {
```

Comment:
I would prefer to still inherit from StdCheats here, for maximum clarity.

Reply:
We can't inherit in both contracts(DeployOptimized, Base_Test), we would get the "Linearization of inheritance graph impossible" error